test/integration/onchain/certificate.test.ts

```diff
@@ -149,7 +149,7 @@ describe('on-chain', function () {
         expect(cert).to.deep.contain({
           id: context.cert.id,
           state: 'issued',
-          embodied_co2: '246914',
+          embodied_co2: '567250',
```

Comment:
This is a big change in this number when I wouldn't have expected any change. This might be related to the shifting times but I want to understand this better

Reply:
indeed! also reverted back